sql/derived_tables/holdings_ext.sql

```diff
@@ -21,6 +21,7 @@ WITH holdings AS (
         json_extract_path_text(data, 'receiptStatus') AS receipt_status,
         json_extract_path_text(data, 'retentionPolicy') AS retention_policy,
         json_extract_path_text(data, 'shelvingTitle') AS shelving_title,
+        json_extract_path_text(data, 'discovery_suppress') AS discovery_suppress,
```

Comment:
Should the JSON field here be referenced as `discoverySuppress` rather than `discovery_suppress`?

Reply:
I ran a test just with discoverySuppress rather than discovery_suppress for items using inventory_items and both worked. I'm not sure why both discoverySuppress and discovery_suppress would work. I just checked the API and it is discoverySuppress. I just made those changes locally. That is so strange that both worked. Sorry about that.

Reply:
Maybe `discoverySuppress` is not present in your test data.  In that case, `json_extract_path_text()` will return `NULL` just as if you used `discovery_suppress`.